### PR TITLE
Ubiquiti airFiber Parser

### PR DESCRIPTION
**Pull request by gt732**
- Added parser for ubiquiti airFiber/airmax radios
- Compliance Tests
- Parser Tests

### CODE REVIEW

**Comment by itdependsnetworks** [comment]
I think the only thing to consider is the name of the OS.

**Comment by gt732** [comment]
> I think the only thing to consider is the name of the OS.

@itdependsnetworks I updated the parser to be the OS of the device "airos". Let me know if you need me to update anything else. Thanks for reviewing!!

**Comment by jeffkala** [comment]
@gt732 is the airfiber not running unifiOS? Just curious if this same parser would work for the whole Ubiquiti family that run unifiOS

**Comment by gt732** [comment]
> @gt732 is the airfiber not running unifiOS? Just curious if this same parser would work for the whole Ubiquiti family that run unifiOS

@jeffkala I don't have any other ubiquiti devices with me so I can't confirm. I work for a WISP so mostly long distance wireless radios. The airfiber/airmax radios from ubiquiti  use a OS called "airOS". The configs for these devices are pretty basic flat files.

File: netutils/config/parser.py [hunk 1]
```diff
@@ -1575,3 +1575,30 @@ def _build_banner(self, config_line: str) -> t.Optional[str]:
...
+    def config_lines_only(self) -> str:
+        """Remove spaces and unwanted lines from config lines.
+
+        Returns:
+            The non-space and non-comment lines from ``config``.
+        """
+        config_lines = []
+        config = self.config.strip()
+        for line in config.splitlines():
+            if line and line != "##" and not self.is_comment(line):
```
Comment (jeffkala) [inline netutils/config/parser.py:1601]: Isn't any number of `#` a comment line?

File: tests/unit/mock/config/parser/base/ubiquiti_airos/airos_full_received.py [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+    ConfigLine(config_line="#radio.1.antenna.1.id=4", parents=()),
+    ConfigLine(config_line="#radio.1.antenna.1.name=Combined", parents=()),
+    ConfigLine(config_line="#radio.1.antenna.1.gain=0", parents=()),
+    ConfigLine(config_line="#radio.1.ccode=840", parents=()),
+    ConfigLine(config_line="#radio.1.ccode.fixed=0", parents=()),
+    ConfigLine(config_line="#radio.1.ccode.locked=1", parents=()),
+    ConfigLine(config_line="#radio.1.ptp_only=1", parents=()),
+    ConfigLine(config_line="#radio.1.ptp_sta=1", parents=()),
+    ConfigLine(config_line="#radio.1.chanbw=100", parents=()),
+    ConfigLine(config_line="#deac9d0212791db5ede667c016876ba4", parents=()),
```
Comment (jeffkala) [inline tests/unit/mock/config/parser/base/ubiquiti_airos/airos_full_received.py:42]: In similar concept to my previous comment, I wouldn't want these in `ConfigLine` as to me I believe these configs are not applied.  I'd expect any line starting with a # is a comment line, whether there is 1,2,or 3 of them. Is this not the case?

Comment (gt732) [reply tests/unit/mock/config/parser/base/ubiquiti_airos/airos_full_received.py]: > 

@jeffkala Sorry for the late reply! Yes correct, I updated the parser to remove the "#" lines. 
Thanks!

Comment (jeffkala) [reply tests/unit/mock/config/parser/base/ubiquiti_airos/airos_full_received.py]: Thanks!

**Review by jeffkala — APPROVED** [review]

**Comment by itdependsnetworks** [comment]
@gt732  can you get tests working all the way through?

**Comment by gt732** [comment]
@itdependsnetworks hey! the black issue should be fixed now.

**Review by itdependsnetworks — APPROVED** [review]

**Pull request merged**